core/shared/src/main/scala/scalaz/zio/stm/STM.scala

```diff
@@ -226,8 +226,14 @@ final class STM[+E, +A] private[stm] (
   /**
    * Maps the success value of this effect to unit.
    */
+  @deprecated("use unit", "1.0.0")
```

Comment:
Can you make this one delegate to the other one, to avoid duplicated code?

Reply:
Removed duplicate code by delegating to new function.